packages/mrm/src/index.js

```diff
@@ -169,7 +172,7 @@ async function runTask(taskName, directories, options, argv) {
 			return;
 		}
 
-		console.log(kleur.cyan(`Running ${taskName}...`));
+		console.log(kleur.cyan(`Running ${taskName} (from preset "${preset}")...`));
```

Comment:
The default UX will be very confusing here. It's going to say `from preset "default"` which is incorrect, we no longer use `mrm-preset-default`. Better to show this only when there's an actual preset used.

Reply:
Fixed. Now it only shows the message when a non-default preset is used.